### ISSUE TITLE
Accuracy #s on the test set?

### ISSUE THREAD

**Issue by avspavan** [KaimingHe/deep-residual-networks#56]
Does anyone have accuracy numbers on Imagenet 50k testset for Resnet-18 and 50? I could find this for validation set but not for test set. My numbers are 65.2 and 72.9, less than the whats seen on the validation set. Also, TF resnet-50 accuracy is similar to Caffe Validation accuracy, strangely. (https://github.com/tensorflow/models/tree/master/research/slim)

My preprocessing for resnet: resize to 256, centercrop 224, resnetmean subraction. Strangely my accuracy is bad if my 1st resize is aspect ration preserving & smaller side to 256. Any comments/suggestions is appreciated. Thanks!

**Comment by susushu** [comment]
hi, i meet the same issues, the top-1 accuracy is also 72.9 of resnet-50, did you find the reason ?
